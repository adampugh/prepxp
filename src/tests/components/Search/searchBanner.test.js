import React from "react";
import { shallow } from "enzyme";

import SearchBanner from "../../../components/Search/searchBanner";

describe("<SearchBanner />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<SearchBanner />);
        expect(wrapper).toMatchSnapshot();
    });
});