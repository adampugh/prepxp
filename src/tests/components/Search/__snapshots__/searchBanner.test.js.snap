// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBanner /> should match snapshot 1`] = `
<div
  className="searchBanner__wrapper"
>
  <div
    className="container-s"
  >
    <h1>
      Search lists
    </h1>
    <p>
      Search job titles for curated lists of interview questions. Click the + symbol to add to your own lists and practise on your dashboard.
    </p>
    <input
      placeholder="Back-End Developer"
      type="text"
    />
    <button
      className="btn"
    >
      Add Question
    </button>
  </div>
</div>
`;
